### PR TITLE
first draft of staging setup

### PR DESCRIPTION
**Pull request by lkolacz**

### CODE REVIEW

File: backend/unpp_api/settings/staging.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+from __future__ import absolute_import
+from .base import *  # noqa: ignore=F403
+
+
+# dev overrides
+DEBUG = False
+IS_STAGING = True
+
+# domains/hosts etc.
+DOMAIN_NAME = os.getenv('DJANGO_ALLOWED_HOST', 'unpp.tivixlabs.com')
```
Comment (nemd) [inline backend/unpp_api/settings/staging.py]: I would change default value from 'unpp.tivixlabs.com' to `localhost`

File: backend/unpp_api/settings/staging.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+WWW_ROOT = 'http://%s/' % DOMAIN_NAME
+ALLOWED_HOSTS = [DOMAIN_NAME, "*"]
```
Comment (nemd) [inline backend/unpp_api/settings/staging.py]: As we talked - asterisk means all domains so it's kind of redundant here with real domain

File: backend/unpp_api/settings/staging.py (hunk 1 above)
Comment (lkolacz) [reply backend/unpp_api/settings/staging.py]: DONE

File: backend/unpp_api/settings/staging.py (hunk 2 above)
Comment (lkolacz) [reply backend/unpp_api/settings/staging.py]: rm .. but please make domain name with dot prefix .. like:
`.unpp.tivixlabs.com`

**Pull request merged**